liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,401 @@
+const {
+  Finding,
+  FindingSeverity,
+  FindingType,
+  ethers,
+  getEthersProvider,
+} = require('forta-agent');
+const BigNumber = require('bignumber.js');
+const config = require('../agent-config.json');
```

Comment:
```suggestion
const config = require('../bot-config.json');
```

Reply:
Updated in commit e08bc045b2f90f2087c0808b37ef836152ae46b4